### DISCUSSION TITLE
Feature request: No code editor for Agolia UI

### DISCUSSION THREAD

**Discussion by Simonheu** [algolia/instantsearch#6405] [Ideas]
I would love to see a no code editor for the UI Algolia UI Builder. The UI Demos Preview in the settings page is a good start, but can of course be taken much further. It would allow less technical people to setup Algolia without the need for devs or experts. Many thanks

**Comment by Haroenv** [comment]
It's on our road map to create a no code editor, starting with collection pages. If there's any ways in which you're imagining this and focus points we can keep in mind while developing, don't hesitate to share them!
